DataFormats/TauReco/src/classes_def_3.xml

```diff
@@ -99,7 +99,13 @@
   <class name="std::pair<reco::PFTauRef, reco::PFTauDecayMode>"/>
   <class name="std::vector<std::pair<reco::PFTauRef, reco::PFTauDecayMode> >" />
 
-  <class name="reco::PFTauTransverseImpactParameter"/>
+  <class name="reco::PFTauTransverseImpactParameter" ClassVersion="11">
+   <version ClassVersion="11" checksum="572950680"/>
+   <version ClassVersion="10" checksum="0000000000"/>
```

Comment:
@Dr15Jones is this line with checksum=0000 meaningful in this case?
I think the "ClassVersion="10" " line should be removed and instead of 11 as a first version number,  2 should be used.
Also, @andrewj314  commented out field should be removed